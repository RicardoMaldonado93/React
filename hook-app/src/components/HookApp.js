import React, { Component } from 'react';

const HookApp = () =>{
 
    return (
        <div className={"container-fluid"}>
         <h1>Hola Mundo</h1>   
        </div>
    )
}

export default HookApp;
