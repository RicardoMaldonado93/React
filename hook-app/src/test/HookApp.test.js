import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import React from 'react';
import HookApp from '../components/HookApp';

describe('Unitary test for HookApp', () => {
   
    test('should do match with snapshot', () => {
        const wrapper = shallow( <HookApp /> );

        expect( wrapper ).toMatchSnapshot();
    });
    
});
