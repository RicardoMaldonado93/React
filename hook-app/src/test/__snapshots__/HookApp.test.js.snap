// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unitary test for HookApp should do match with snapshot 1`] = `
<div
  className="container-fluid"
>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
